### PR TITLE
[eck-operator]: Additional Environment Variables

### PR DESCRIPTION
**Pull request by oliverbaehler**
Signed-off-by: Oliver Bähler <oliverbaehler@hotmail.com>

With this PR the possibility to add custom environment variables to the operator Container.  We currently are required to set a proxy to have internet access with the operator. This would be possible with e.g. the following configuration:

```
env:
  - name: "HTTP_PROXY"
    value: "https://company-proxy:8000"
  - name: "HTTPS_PROXY"
    value: "https://company-proxy:8000"
```

**Note**: I wasn't quiet sure how to bump the chart version. Please let me know how to bump it if that change is ok with you. Second my Editor has removed some spaces resulting in a lot more diffs than there are actual changes. Let me know if i should readd the spaces.

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/cloud-on-k8s/tree/master/CONTRIBUTING.md)?
- If you submit code, is your pull request against master? We recommend pull requests against master. We will backport them as needed.

### CODE REVIEW

**Comment by pebrc** [comment]
Jenkins test this please

File: deploy/eck-operator/values.yaml [hunk 1]
```diff
@@ -54,7 +54,10 @@ nodeSelector: {}
 tolerations: []
 
 # affinity defines the node affinity rules for the operator pod.
-affinity: {} 
+affinity: {}
+
+# additional environment variables
```
Comment (charith-elastic) [inline deploy/eck-operator/values.yaml:59]:
```suggestion
# additional environment variables for the operator container.
```

**Review by charith-elastic — COMMENTED** [review]
Thanks for the PR. LGTM apart from a nit about the comment explaining the purpose of `env`.

**Comment by charith-elastic** [comment]
Jenkins test this please

**Review by charith-elastic — APPROVED** [review]

**Pull request merged**